### DISCUSSION TITLE
how to use ctrl shift and esc for xkill

### DISCUSSION THREAD

**Discussion by jobswelldone** [i3/i3#5856] [Q&A] [answered]
i was using this keybind  
```
bindsym Control_L+Shift_L+Escape exec  --no-startup-id xkill
```
but it doesnot resigter it .
i have also tried 

```
bindsym Control+Shift+Escape exec  --no-startup-id xkill
```

please help

**Comment by orestisfl** [comment]
Try `bindsym --release`

**Reply by jobswelldone** [reply]
no luck

**Reply by orestisfl** [reply, marked as answer]
`bindsym --release Control+Shift+Escape exec --no-startup-id xkill` works fine for me.

1. Make sure to apply the changes in the i3 config file with `i3-msg reload`
2. Make sure that `xkill` is installed and works when you try it from the command line
3. Use `bindsym --release Control+Shift+Escape exec --no-startup-id xkill &> /tmp/log` to see if there are any error messages from `xkill`
4. If that all fails please share your i3 logs, see https://i3wm.org/docs/debugging.html
